### DISCUSSION TITLE
2.11.0

### DISCUSSION THREAD

**Discussion by TheLastProject** [CatimaLoyalty/Android#625] [Releases]
- Add Catima to [Quick Access Device Controls](https://developer.android.com/guide/topics/ui/device-control)
- Fix some groups not showing up correctly in group management screen

<hr /><em>This discussion was created from the release <a href='https://github.com/TheLastProject/Catima/releases/tag/v2.11.0'>2.11.0</a>.</em>

**Comment by CelianGdfrd** [comment]
OMG this feature really will help me use Catima ! Thanks a lot !!
